flink-ml-lib/src/test/java/org/apache/flink/ml/feature/StandardScalerTest.java

```diff
@@ -206,7 +206,7 @@ public void testSaveLoadAndPredict() throws Exception {
         model = TestUtils.saveAndReload(tEnv, model, tempFolder.newFolder().getAbsolutePath());
 
         assertEquals(
-                Arrays.asList("mean", "std"),
+                Arrays.asList("mean", "std", "version", "timestamp"),
```

Comment:
Well, I think letting `StandardScalerModel` have model version and timestamp is reasonable, since offline learning could be viewed as a special case of online learning. What we need to do is to make sure that version and timestamp has a clear meaning. For example, the version of offline model is always zero and time stamp is `LONG.MAX_VALUE`.

What do you mean by verify `model data contains those columns`? Do you mean we only verify `model data` and leave `model version` and `timestamp` unchecked?

Reply:
> What do you mean by verify model data contains those columns? Do you mean we only verify model data and leave model version and timestamp unchecked?

Yes that is what I means. IMO StandardScalerTest should verify the expected behavior of StandardScalerModel from the perspective of users who want to use StandardScalerModel. IMO users of StandardScalerModel should not need to know concepts such as model version or timestamp. Users should `OnlineStandardScaler` needs to know these information, and this behavior is covered by `OnlineStandardScalerTest`.

> I think letting StandardScalerModel have model version and timestamp is reasonable, since offline learning could be viewed as a special case of online learning

Yes we can tell users this. But for users of StandardScaler, there is no benefit to knowing this. So it seems simpler not to tell users of `StandardScalerModel` about version etc.?

Reply:
Thanks for the explanation. I have updated the PR as you suggested.